.eslintrc.json

```diff
@@ -2,33 +2,30 @@
     "root": true,
     "env": {
         "browser": true,
-        "es2017": true,
+        "es2018": true,
```

Comment:
Why is this bumped?

Reply:
At the time the config was introduced, it only had options for es versions that introduced new globals. It seems this changed in an eslint update from last year: https://eslint.org/blog/2022/02/eslint-v8.9.0-released/#highlights

>Previously, we used to add new ES environments only if they contain new globals compared to previous years. New es2016, es2018, and es2019 environments have also been added to fill in the gaps between years.

So this just aligns it to what was initially intended: ES9 (2018)